### DISCUSSION TITLE
Include JS Asset: "Identifier X has already been declared"

### DISCUSSION THREAD

**Discussion by flolanger** [filamentphp/filament#11339] [Help]
### Package

Panel builder

### Package Version

3.2.25

### How can we help you?

I'm trying to implement the sentry sdk to my filament project.

If I include the JavaScript like the filament docs say, it doesn't bundle it through vite, so this doesn't work:
```php
        FilamentAsset::register([
            Js::make('sentry',  __DIR__ . '/../../resources/js/sentry.js'),
        ]);
```

If I use the Vite facade it does load the correct file (bundled from vite), but I get "Identifier X has already been declared", because vite doesn't know, that there are other assets as well, like filament core and other filament plugins.
```php
        FilamentAsset::register([
            Js::make('sentry', Vite::asset('resources/js/sentry.js')),
        ]);
```

How can I avoid this problem?

Thank you!

**Comment by leandrocfe** [comment]
Is there a compiled version or cdn that you can register it?

**Reply by flolanger** [reply]
No, because it includes settings and configuration.
